src/AzureStack/CommerceAdmin/Commerce.Admin/generate.cmd

```diff
@@ -4,6 +4,7 @@
 ::
 
 @echo off
-call %~dp0..\..\..\..\tools\generate.cmd azsadmin/resource-manager/commerce latest Azure current azure-rest-api-specs %CD%
+call %~dp0..\..\..\..\tools\generate.cmd azsadmin/resource-manager/commerce latest Azure bcb638a668dbb79eaf3190324ca6d690be84b9f6 azure-rest-api-specs %CD%
```

Comment:
Branch should be master instead of a commit id here.

Reply:
Done.